MonolithDev/gettingSongs/test_dj_queries.py

```
@@ -0,0 +1,187 @@
+        
+        # Test processing individual tracks
+        for i, track in enumerate(sample_tracks[:2], 1):  # Test first 2 tracks
+            print(f"\n{'-'*60}")
+            print(f"PROCESSING TRACK {i}")
+            print(f"{'-'*60}")
+            
+            try:
+                result = finder.process_track(track)
+                finder.print_track_queries(result)
+                
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)